reinforcement_learning/unit_test/event_test.cc

```diff
@@ -71,10 +75,14 @@ BOOST_AUTO_TEST_CASE(serialize_empty_ranking)
   ranking.set_model_id("model_id");
 
   utility::data_buffer oss;
+<<<<<<< HEAD
```

Comment:
This won't compile

Reply:
this file is not included in the build and should be deleted. As there it will be replaced by flatbuffer and flatbuffer has its own unit tests for this case.